### PR TITLE
Update RouterController.php

### PR DESCRIPTION
**Pull request by htetwunsan**
Added unique:routers,ip validator rule to create router. Issue#654

### CODE REVIEW

File: app/Http/Controllers/Network/RouterController.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function store(Request $request)
         $this->authorize('create', Router::class);
 
         Validator::make($request->all(), [
-            'ip' => 'required|max:15|ip',
+            'ip' => 'required|max:15|unique:routers,ip',
```
Comment (kdmnk) [inline app/Http/Controllers/Network/RouterController.php:45]: 'ip' is a separate rule, we need: '..|unique:routers,ip|ip'

Comment (kdmnk) [reply app/Http/Controllers/Network/RouterController.php:45]: And please add the same for updates, on line 70

**Review by kdmnk — CHANGES_REQUESTED** [review]
Hi, thanks, I left some comments.

**Review by kdmnk — APPROVED** [review]
thank you

**Comment by htetwunsan** [comment]
Hey, i missed database name in rule. I created another pull request for that.

**Pull request merged**